lib/tasks/search.rake

```diff
@@ -1,170 +1,36 @@
 require 'task_helpers'
 
 namespace :search do
-  def import_from_cursor(cursor, index, opts)
-    tot = cursor.count
-    cnt = 0
-    t = Time.now
-    index.import cursor, {:method => :paginate, :per_page => opts[:batch_size]} do |documents|
```

Comment:
Only load testing will show if the defaults are good enough.  We may not even use this for the migration.  Do we want to load test this anyway since it exists, just in case someone wants to run this in the future?
FYI: @maxrothman